packages/metadata-common/src/AddMetadataButton.tsx

```diff
@@ -110,14 +106,26 @@ export const AddMetadataButton = (
             <MenuList id="split-button-menu">
               {sortedSchema?.map((schema: IDictionary<any>) => (
                 <MenuItem
-                  key={schema.display_name}
-                  title={`New ${schema.display_name} ${props.schemaType ?? ''}`}
+                  key={schema.title}
+                  title={`New ${schema.title} ${
+                    schema.title
+                      .toLowerCase()
+                      .includes(props.schemaType?.toLocaleLowerCase())
+                      ? ''
+                      : props.schemaType
```

Comment:
In case of the new metadata options displayed when clicking the Add new metadata button, we only append `schemaType` info if it's not already included in the schema title, avoiding repetitions such as
![image](https://user-images.githubusercontent.com/25207344/140230980-d847ca0c-0b87-45cc-8940-0cde59b07d9d.png)